client/hostinfo_unix.cpp

```diff
@@ -2003,6 +2010,75 @@ const vector<string> X_display_values_initialize() {
     return display_values;
 }
 
+void getIdleTime() {
+    long idleTime = 0;
+    // Use glob to enumerate input devices in /dev/input/
+    glob_t globbuf;
+    if (glob("/dev/input/event*", GLOB_NOSORT, nullptr, &globbuf) != 0) {
+        std::cerr << "Failed to enumerate input devices. " << std::endl;
```

Comment:
`msg_printf`should be used instead of the direct `std::` calls.

Reply:
Please fix this comment as well

Reply:
It should be all good now.